### PR TITLE
Update Badge 4 TT grader

### PR DESCRIPTION
**Pull request by Astri-Cornish**
Change json deserialiser to json_to_QuasiDistribution since Q-Ds are used here instead of regular prob distributions

### CODE REVIEW

**Comment by Astri-Cornish** [comment]
Tagging @quantum-kittens for visibility as you're not coming up in the reviewers list

**Review by vabarbosa — APPROVED** [review]
thank you @Astri-Cornish 
looks good to me. once i get an additional approval from @HuangJunye or @quantum-kittens i will merge

**Pull request merged**
